cns/service/main.go

```diff
@@ -865,7 +865,7 @@ func reconcileInitialCNSState(ctx context.Context, cli nodeNetworkConfigGetter,
 	}
 
 	// Convert to CreateNetworkContainerRequest
-	ncRequest, err := kubecontroller.CRDStatusToNCRequest(&nnc.Status)
+	ncRequest, err := kubecontroller.CreateNCRequestFromDynamicNC(nnc.Status.NetworkContainers[0]) // swift will only ever support 1 NC
```

Comment:
>// swift will only ever support 1 NC [](http://example.com/codeflow?start=96&length=36)

This may not be true in future.

Can you update the comment " For now AKS Swift supports 1 NC"

Reply:
updated to support multiple NCs